tests/test_middleware.py

```diff
@@ -1,191 +0,0 @@
-from unittest.mock import Mock, patch
```

Comment:
The tests related to startup were moved to test_startup and the ones related to open_session moved to test_open_session.